packages/cydran/src/log/LoggerImpl.ts

```diff
@@ -42,7 +42,7 @@ class LoggerImpl implements Logger {
 	constructor(context: Context, appender: Appender, key: string, label: string) {
 		this.appender = requireNotNull(appender, "appender");
 		this.key = requireNotNull(key, "key");
-		this.label = label;
+		this.label = label ?? "-- unknown --";
```

Comment:
The defaulted() method in Utils.ts handled both null and undefined, whereas I believe that this syntax only handles one of the two.

Reply:
done